atlas_anndata/funcs.py

```diff
@@ -967,7 +967,7 @@ def write_cell_metadata(
     precells_filename = f"mage-tab/{exp_name}.precells.txt"
 
     cell_metadata, run_metadata, cell_specific_metadata = derive_metadata(
-        adata, config=config, kind=None
+        adata, config=config, write_premagetab=write_premagetab, kind=kind
```

Comment:
If you choose to set `write_premagetab` to `True` as default, this change is not needed.